indigo_app/templates/library.html

```diff
@@ -37,74 +37,52 @@
   <thead>
     <tr>
       <th class="title" data-sort="title">Title</th>
-      <th class="year" data-sort="year">Year / Number</th>
-      <th class="updated_at" data-sort="updated_at">Updated</th>
+      <th class="uri" data-sort="uri">Frbr uri</th>
+      <th class="langs" data-sort="langs">Languages</th>
+      <th class="psit" data-sort="psit">Points in time</th>
     </tr>
   </thead>
+
   <tbody>
     {{#each works}}
-      {{#unless docs}}
-        <tr>
-          <!-- work without any documents -->
-          <td>
+      <tr>
+
+        {{!-- Title --}}
+        <td>
+          {{#if work_docs}}
+            <i class="fa fa-fw fa-circle published" title="published"></i>
+          {{else}}
             <i class="fa fa-fw fa-circle" title="empty work"></i>
-            <a href="/works{{ frbr_uri }}/">{{ title }}</a>
-            {{#if repealed}}
-              <span class="badge badge-info">repealed</span>
-            {{/if}}
-          </td>
-          <td>{{ year }} / {{ number }}</td>
-          <td>
-            <span class="time-ago" data-timestamp="{{ updated_at }}">{{ updated_at }}</span>
-            <span class="text-muted">{{#if updated_by_user.display_name}}- {{ updated_by_user.display_name }}{{/if}}</span>
-          </td>
-        </tr>
-      {{/unless}}
-
-      {{#each docs}}
-        <tr {{#unless @first}}class="amended-version"{{/unless}}>
-          <td>
-            {{#if draft}}
-              <i class="fa fa-fw fa-circle draft" title="draft"></i>
-            {{else}}
-              <i class="fa fa-fw fa-circle published" title="published"></i>
-            {{/if}}
-
-            {{#if @first}}
-              <a href="/documents/{{ id }}/">{{ title }}</a>
-              <span class="text-muted">as at {{ expression_date }}</span>
-            {{else}}
-              <a href="/documents/{{ id }}/">{{ title }} as at {{ expression_date }}</a>
-            {{/if}}
-
-            {{#if ../repealed_date}}
-              <span class="badge badge-info">repealed</span>
-            {{/if}}
-
-            {{#if stub}}
-              <span class="badge badge-info">stub</span>
-            {{/if}}
-
-            {{#each tags}}
-              <span class="badge badge-secondary">{{this}}</span>
+          {{/if}}
+          <a href="/works{{ frbr_uri }}/">{{ title }}</a>
+        </td>
+
+        {{!-- Frbr uri --}}
+        <td>
+          <a href="/works{{ frbr_uri }}/">{{ frbr_uri }}</a>
+        </td>
+
+        {{!-- Languages --}}
+        <td>
+            {{#each languages}}
+            • {{this}}
```

Comment:
You can use `@first` or `@last` in the loop to test if this is the first or last iteration in the loop, to ensure the middot only shows up between items.

https://handlebarsjs.com/builtin_helpers.html

``{{#if @first}} ... {{/if}}``